jpo-geojsonconverter/src/test/resources/json/sample.ode-tim.json

```diff
@@ -0,0 +1,451 @@
+{
+	"metadata": {
+		"logFileName": "",
+		"recordType": "timMsg",
+		"securityResultCode": "success",
+		"receivedMessageDetails": {
+			"rxSource": "NA"
+		},
+		"payloadType": "us.dot.its.jpo.ode.model.OdeMessageFramePayload",
+		"serialId": {
+			"streamId": "aff21add-4305-4007-b25d-a9de2cb9441c",
+			"bundleSize": 1,
+			"bundleId": 0,
+			"recordId": 0,
+			"serialNumber": 0
+		},
+		"odeReceivedAt": "2025-10-07T21:40:19.711Z",
+		"schemaVersion": 9,
+		"maxDurationTime": 0,
+		"recordGeneratedAt": "",
+		"recordGeneratedBy": "RSU",
+		"sanitized": false,
+		"odePacketID": "",
+		"odeTimStartDateTime": "",
+		"asn1": "001F81047015E5D88D442EF003FC6B1B040F775D9B1B0100003F42F2EC21C0A001E84E2C61ED2CE9F09D2711842142491D01225EFA874512D7542DE894716125C4AEB703D410E360C408617000000C80218313711081EEEBB3630100003F42F2EC21C0A003EA00000000271630F69674F84E9388C3F0A1248E80912F7D43A2896BAA16F44A38B092E2575B81EA0871B0620430B80000064010C189B88840F775D9B18080001FA44C86384050019500000000138AFB180B39DA36C43FFFD4E2BEC602CE768DB57400FA60000013043DDD766C40400007E96363C01684003729C57FF3259D166423CE805BBFE1E0022DE36BD9314EE99FDFCB01C6A97C5C41A81A4000010D820502C0",
+		"source": "RSU",
+		"originIp": "172.27.0.1",
+		"isCertPresent": false
+	},
+	"payload": {
+		"data": {
+			"messageId": 31,
+			"value": {
+				"TravelerInformation": {
+					"msgCnt": 1,
+					"timeStamp": 386520,
+					"packetID": "8D442EF003FC6B1B04",
+					"urlB": "null",
+					"dataFrames": [
+						{
+							"doNotUse1": 0,
+							"frameType": "advisory",
```

Comment:
suggestion: following the CTW guidance, we should not be setting this to advisory. Since all CTW group members are public road authorities they will be using "roadSignage", with "advisory" being marked as "do not use" going forward.

Reply:
Updated to use sample TIMs used at plugfest that don't use `"frameType": "advisory"`